wordpress-mobile/WordPress-Android
# Replace Volley with Glide in the EditPostSettings fragment

Fixes #7967

Replaces Volley with Glide in the EditPostSettings.

Adds support for adding animated feature images.

MaxHeight attribute of the ImageView is taken into account - it was ignored before.

To test:

1. Click on the create post button
3. Type any title
5. Click on three dots in the top right corner of the screen
7. Click on Post Settings
9. Click on the Set feature image button
11. Select any GIF image
13. Notice the animation is working and click on the Publish button
15. Go to Reader - Followed Sites
17. Make sure the post has animated GIF image
